templates/add-ons/documentation_starter/{{ cookiecutter.__folder_name }}/Makefile

```
@@ -52,38 +50,38 @@ clean-python: clean
 # documentation builders
 .PHONY: html
 html: .venv/bin/python  ## Build html
-	cd $(DOCS_DIR) && $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	cd \$\(DOCS_DIR\) \&\& \$\(SPHINXBUILD\) -b ([^ ]+) \$\(ALLSPHINXOPTS\) \$\(BUILDDIR\)/([^ ]+)
```

I don't understand this change. I went with this in Fluffy.

```suggestion
	@uv run sphinx-build -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```